packages/html/stories/Editing.stories.js

```diff
@@ -150,7 +150,7 @@ const Template = ({ label, ...args }) => {
       value,
       position: [100, 60],
       size: [120, 80],
-      style: { overflow: 'fill' },
+      style: { overflow: 'fill', editable: true },
```

Comment:
**note:** Once #570 has been merged, this should no longer be necessary.

Reply:
#570 has been merged, so merging `main` into the branch of this PR should allow to remove the "editable" configuration

Reply:
Updated this branch to be up to date with main, works properly without having to add editable:true when inserting vertex.